pkg/testconfig/s3testconfigs/s3_test_configs.go

```diff
@@ -4,40 +4,40 @@ package s3testconfigs
 import (
 	"os"
 
+	"github.com/spf13/viper"
+
 	"github.com/cms-enterprise/mint-app/pkg/appconfig"
 	"github.com/cms-enterprise/mint-app/pkg/s3"
-	"github.com/cms-enterprise/mint-app/pkg/testhelpers"
 )
 
 // S3TestClient returns an S3Test client for testing
-func S3TestClient() s3.S3Client {
-	config := testhelpers.NewConfig()
+func S3TestClient(viperConfig *viper.Viper) s3.S3Client {
 
 	s3Cfg := s3.Config{
-		Bucket:  config.GetString(appconfig.AWSS3FileUploadBucket),
-		Region:  config.GetString(appconfig.AWSRegion),
+		Bucket:  viperConfig.GetString(appconfig.AWSS3FileUploadBucket),
+		Region:  viperConfig.GetString(appconfig.AWSRegion),
 		IsLocal: true,
 	}
 	//OS ENV won't get environment variables set by VSCODE for debugging
-	_ = os.Setenv(appconfig.LocalMinioAddressKey, config.GetString(appconfig.LocalMinioAddressKey))
-	_ = os.Setenv(appconfig.LocalMinioS3AccessKey, config.GetString(appconfig.LocalMinioS3AccessKey))
-	_ = os.Setenv(appconfig.LocalMinioS3SecretKey, config.GetString(appconfig.LocalMinioS3SecretKey))
+	_ = os.Setenv(appconfig.LocalMinioAddressKey, viperConfig.GetString(appconfig.LocalMinioAddressKey))
+	_ = os.Setenv(appconfig.LocalMinioS3AccessKey, viperConfig.GetString(appconfig.LocalMinioS3AccessKey))
+	_ = os.Setenv(appconfig.LocalMinioS3SecretKey, viperConfig.GetString(appconfig.LocalMinioS3SecretKey))
```

Comment:
Same question as before with regards to the `os.Setenv` lines -- can't quite make out why these are needed from reading through this